animate_training.py

```diff
@@ -148,13 +94,13 @@ def main(hidden, width, activation, nepoch, lr, funcname, xdomain, numtrains, de
     # inherit/modify the get_tr_results for different training
     # pass in Trainer(name, xt, yt) for full control
     train.optimkwargs["lr"] = lr
-    activationfunc = get_activation(activation)  # get's relu by default
+    activationfunc = swarm.get_activation(activation)  # get's relu by default
     # end config
     data_list = []
     print("Starting training")
     tr_start = pendulum.now()
     for i in range(numtrains):
-        net = nn.Sequential(*make_net(hidden, width, activationfunc))
+        net = networks.flat_net(hidden, width, activationfunc)
         data, loss = train.get_training_results(net, nepoch)
```

Comment:
These were the lines that ran into trouble for me when I tried to run on the command line. The output is in slack.

Reply:
should be fixed now